pkg/agent/multicast/mcast_controller.go

```diff
@@ -82,12 +82,14 @@ type GroupMemberStatus struct {
 	group net.IP
 	// localMembers is a map for the local Pod member and its last update time, key is the Pod's interface name,
 	// and value is its last update time.
-	localMembers map[string]time.Time
+	localMembers      map[string]time.Time
```

Comment:
I don't think it is necessary to introduce Mutexes for localMembers and remoteMembers. The root cause of the issue is we didn't create a new object for remoteMembers before updating the cache. The change in line 137 is enough.